src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,167 @@
+import { HTMLAttributes, ReactNode, useId } from "react";
+import styled from "styled-components";
+import { IconButton, Label } from "@/components";
+import { Error, FormElementContainer, FormRoot } from "../commonElement";
+
+interface CommonProgressBarProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, "children"> {
+  progress: number;
+  label?: ReactNode;
+  error?: ReactNode;
+  orientation?: "vertical" | "horizontal";
```

Comment:
this is a bit misleading. I would expect this refers to the progress bar orientation. Maybe let's call `labelOrientation`?

Reply:
Currently we are using orientation in all the form elements
Thats why I added in a similar fashion